eclair-core/src/main/scala/fr/acinq/eclair/channel/ChannelTypes.scala

```diff
@@ -309,7 +309,7 @@ final case class LocalParams(nodeId: PublicKey,
                              maxAcceptedHtlcs: Int,
                              isFunder: Boolean,
                              defaultFinalScriptPubKey: ByteVector,
-                             staticPaymentBasepoint: Option[PublicKey],
+                             walletStaticPaymentBasepoint: Option[PublicKey],
```

Comment:
It would be misleading otherwise. When using anchor outputs, we do use a static payment basepoint (since we use `static_remotepubkey`) but it's not the bitcoin core wallet that hold the key.
With that renaming I want to stress that this is `Some(...)` only when the static payment basepoint is from our bitcoin wallet, does that make sense?